api/Dockerfile

```diff
@@ -8,6 +8,7 @@ RUN apt-get update && \
     curl \
     libpq-dev \
     pkg-config \
+    postgresql \
```

Comment:
Had to add this so we can use the `psql` shell in the CI. It's super small, less than 1MB.